### ISSUE TITLE
An ADA issue: the «MY CART» title in the top-right corner has an insufficient color contrast

### ISSUE THREAD

**Issue by dmitry-fedyuk** [kitchencollection-com/core#6]
[kitchencollection.com](https://www.kitchencollection.com)
[developer.paciellogroup.com/resources/contrastanalyser](https://developer.paciellogroup.com/resources/contrastanalyser)

![05](https://user-images.githubusercontent.com/83204/46411029-88affc00-c73c-11e8-8e31-16fd0bf752db.png)

![01](https://user-images.githubusercontent.com/83204/46411089-b2692300-c73c-11e8-942f-a1e623153a6a.png)

**Comment by dmitry-fedyuk** [comment]
The parent task: [An ADA issue: «7. Insufficient color contrast»](https://github.com/kitchencollection-com/core/issues/3)

**Comment by dmitry-fedyuk** [comment]
Fixed in [0.0.4](https://github.com/kitchencollection-com/core/releases/tag/0.0.4).
